pyproject.toml

```diff
@@ -77,7 +88,7 @@ exclude = '''
 [tool.coverage.run]
 parallel = true
 branch = true
-source = ["lsstcst"]
+source = ["lsst"]
```

Comment:
I think this makes coverage of all lsst, including imported files lsst.*, is better to do lsst.cst?

Reply:
changed to `lsst.cst`